### PR TITLE
Update the audio fusion result with a better tsn-f3 result.

### LINKED ISSUE #280: The performance of fusing RGB & Audio needs to be updated.

### PR DESCRIPTION
**Pull request by SuX97**
Fixes #280

### CODE REVIEW

File: configs/recognition_audio/resnet/README.md [hunk 1]
```diff
@@ -61,3 +61,13 @@ python tools/test.py configs/audio_recognition/tsn_r50_64x1x1_100e_kinetics400_a
 ```
 
 For more details, you can refer to **Test a dataset** part in [getting_started](/docs/getting_started.md#test-a-dataset).
+
+## Fusion
+For multi-modality fusion, you can use the simple [script](/tools/analysis/report_accuracy.py), the standard usage is:
+
+```shell
+python tools/analysis/report_accuracy.py --scores ${AUDIO_RESULT.PKL} ${VISUAL_RESULT.PKL}   --datalist data/kinetics400/kinetics400_val_list_rawframes.txt  --coefficient 1 1
```
Comment (dreamerlin) [inline configs/recognition_audio/resnet/README.md:69]: single blank

**Review by dreamerlin — APPROVED** [review]

File: configs/recognition_audio/resnet/README.md [hunk 2]
```diff
@@ -14,10 +14,10 @@
...
-|[tsn_r18_64x1x1_100e_kinetics400_audio_feature](/configs/recognition_audio/resnet/tsn_r18_64x1x1_100e_kinetics400_audio_feature.py) + [tsn_r50_1x1x3_100e_kinetics400_rgb](/configs/recognition/tsn/tsn_r50_1x1x3_100e_kinetics400_rgb.py)|1024|8| ResNet(18+50) | None |70.01|88.71|x|x|x|x|x|
+|[tsn_r18_64x1x1_100e_kinetics400_audio_feature](/configs/recognition_audio/resnet/tsn_r18_64x1x1_100e_kinetics400_audio_feature.py) + [tsn_r50_1x1x3_100e_kinetics400_rgb](/configs/recognition/tsn/tsn_r50_1x1x3_100e_kinetics400_rgb.py)|1024|8| ResNet(18+50) | None |71.50(+0.39)|90.18(+0.14)|x|x|x|x|x|
```
Comment (kennymckormick) [inline configs/recognition_audio/resnet/README.md:20]: Please fix the RGB config link, the RGB predictions are given by the model: tsn_r50_video_320p_1x1x3_100e_kinetics400_rgb

**Review by kennymckormick — COMMENTED** [review]
Remember to add a line in the changelog

**Comment by SuX97** [comment]
> Remember to add a line in the changelog

Updated.

**Review by kennymckormick — APPROVED** [review]

**Comment by SuX97** [comment]
ping @dreamerlin , ready to merge.

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
need to rebase

**Comment by SuX97** [comment]
> need to rebase

Rebased.

**Pull request merged**